data/schemas/com.github.akiraux.akira.gschema.xml.in

```diff
@@ -60,6 +60,12 @@
             <description>Allow user to choose between normal and symbolic icons in HeaderBar.</description>
         </key>
 
+        <key name="invert-sidebar" type="b">
+            <default>false</default>
+            <summary>Invert Sidebars Order</summary>
+            <description>Allow user to choose between default and inverted sidebars order.</description>
```

Comment:
Rename this with: "Invert Panels Order", since we will refer to them as Options Panel and Layers Panel in the upcoming documentation.
Update this term accordingly everywhere else you added strings.